### ISSUE TITLE
Paper Spigot 1.15 Startup error

### ISSUE THREAD

**Issue by leasoncre** [JoelGodOfwar/MoreMobHeads#1]
Paper Spigot 1.15.1, Java 8u232 (Linux-Debian 64bit)
looks like a typo in the code with the use of "\\" and "/" where not needed. the file does exist.

Startup error:
```
[20:02:14] [Server thread/INFO]: [MoreMobHeads] Enabling MoreMobHeads v1.14_1.0.0
[20:02:14] [Server thread/INFO]: [MoreMobHeads] plugins/MoreMobHeads/config.yml
[20:02:14] [Server thread/WARN]: java.io.FileNotFoundException: plugins/MoreMobHeads\lang\/en_US.yml (No such file or directory)
[20:02:14] [Server thread/WARN]: 	at java.io.FileInputStream.open0(Native Method)
[20:02:14] [Server thread/WARN]: 	at java.io.FileInputStream.open(FileInputStream.java:195)
[20:02:14] [Server thread/WARN]: 	at java.io.FileInputStream.<init>(FileInputStream.java:138)
[20:02:14] [Server thread/WARN]: 	at org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:126)
[20:02:14] [Server thread/WARN]: 	at com.github.joelgodofwar.mmh.MoreMobHeads.onEnable(MoreMobHeads.java:98)
[20:02:14] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:263)
[20:02:14] [Server thread/WARN]: 	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:338)
[20:02:14] [Server thread/WARN]: 	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:420)
[20:02:14] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugin(CraftServer.java:468)
[20:02:14] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_15_R1.CraftServer.enablePlugins(CraftServer.java:382)
[20:02:14] [Server thread/WARN]: 	at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:481)
[20:02:14] [Server thread/WARN]: 	at net.minecraft.server.v1_15_R1.DedicatedServer.init(DedicatedServer.java:290)
[20:02:14] [Server thread/WARN]: 	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:884)
[20:02:14] [Server thread/WARN]: 	at java.lang.Thread.run(Thread.java:748)
[20:02:14] [Server thread/INFO]: [33m**************************************[0m
[20:02:14] [Server thread/INFO]: [32mMoreMobHeads v1.14_1.0.0[0m is Enabled
[20:02:14] [Server thread/INFO]: [33m**************************************[0m
```

**Comment by JoelGodOfwar** [comment]
Expect a fix tomorrow.

**Comment by JoelGodOfwar** [comment]
Well it seems my lack of knowledge about unix based systems, has led me to use the \ fileseperator hardcoded, and that will cause errors on unix based systems. This is being fixed, and new versions will be rolled out today to fix this on all MC versions.

**Comment by JoelGodOfwar** [comment]
Fixed in v1.14_1.0.2
